Signum.React.Extensions/Dynamic/View/FindOptionsComponent.tsx

```diff
@@ -609,7 +609,7 @@ class ColumnOptionsComponent extends BaseOptionsComponent<ColumnOptionExpr> {
         <td>
           <QueryTokenBuilderString label="columnName" parsedToken={item.parsedToken} token={item.token}
             onChange={newToken => this.handleColumnChange(item, newToken)}
-            queryKey={this.props.queryKey} subTokenOptions={SubTokensOptions.CanElement} hideLabel={true} />
+            queryKey={this.props.queryKey} subTokenOptions={SubTokensOptions.CanElement | SubTokensOptions.CanOperation} hideLabel={true} />

```

Comment:
Is OperstionButton compatible with aggregates/groupby. 

I think in the current implementation is not so better to disable it in this case and the ones below.

Reply:
> Is OperstionButton compatible with aggregates/groupby.
> I think in the current implementation is not so better to disable it in this case 

No it's not compatible, (actually I didn't considered this is for aggregates/groupby 😃 )
I'll revert this one

> and the ones below.

but this is the only one I've added in this file (in its ColumnOptions), no others, right?